src/backend/tests/projects.test.ts

```diff
@@ -666,4 +666,20 @@ describe('Projects', () => {
       expect(prisma.material_Type.create).toBeCalledTimes(1);
     });
   });
+
+  describe('Deleting material type', () => {
+    test('Delete Material Type does not work', async () => {
+      await expect(ProjectsService.deleteMaterialType('NERSoftwareTools', theVisitor)).rejects.toThrow(
+        new AccessDeniedException('Only an admin or head can delete a material type')
+      );
+    });
+
+    test('Delete Material Type works', async () => {
+      vi.spyOn(prisma.material_Type, 'findUnique').mockResolvedValue(toolMaterial);
+      vi.spyOn(prisma.material_Type, 'delete').mockResolvedValue(toolMaterial);
+      const deletedMaterialType = await ProjectsService.deleteMaterialType('NERSoftwareTools', superman);
+      expect(deletedMaterialType.name).toBe('NERSoftwareTools');
+      expect(prisma.material_Type.delete).toBeCalledTimes(1);
+    });
+  });
```

Comment:
- I would suggest changing the name of the first test to specify why it throws (because of permission)
- Can you add another test to ensure that the service throws when the material type does not exist?